File: samples/java_springboot/54.teams-task-module/src/main/java/com/microsoft/bot/sample/teamstaskmodule/models/UISettings.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.bot.sample.teamstaskmodule.models;

public class UISettings {
    int width;
    int height;
    String title;
    String id;
    String buttonTitle;

    public UISettings(
        int withWidth,
        int withHeight,
        String withTitle,
        String withId,
        String withButtonTitle
    ) {
        width = withWidth;
        height = withHeight;
        title = withTitle;
        id = withId;
        buttonTitle = withButtonTitle;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getButtonTitle() {
        return buttonTitle;
    }

    public void setButtonTitle(String buttonTitle) {
        this.buttonTitle = buttonTitle;
    }
}

File: archive/samples/csharp_dotnetcore/54.teams-task-module/Models/UISettings.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.BotBuilderSamples.Models
{
    public class UISettings
    {
        public UISettings(int width, int height, string title, string id, string buttonTitle)
        {
            Width = width;
            Height = height;
            Title = title;
            Id = id;
            ButtonTitle = buttonTitle;
        }

        public int Height { get; set; }
        public int Width { get; set; }
        public string Title { get; set; }
        public string ButtonTitle { get; set; }
        public string Id { get; set; }
    }
}
